### ISSUE TITLE
DeepLab setting 

### ISSUE THREAD

**Issue by VanHulk** [halbielee/EPS#10]
Fellow your step, I get the mIOU about 68% which is much lower than you put on the git(71%).I check out all the steps and can't find any change that I possibly make that might influence the perfermance. Would you tell me how to achieve the ~71% mIOU on VOC . Maybe I miss out some key training strategies.

**Comment by halbielee** [comment]
Hello VanHulk!

We follow the code from [deeplab-pytorch](https://github.com/kazuto1011/deeplab-pytorch) for the segmentation setting and we do not change anything except the hyperparameter for the training. Maybe it is because of the setting for the CRF. I

Please let me know the setting you used for the training and the performance of the pseudo-masks. 
In this repo the trained network for EPS is not the one we used in the paper, so it can be the differernce.

**Comment by VanHulk** [comment]
Hello halbielee！
I follow your code to generate the pseudo-masks without any change and the accuracy(mIOU) is 69.3%. Just now, I get the mIOU  70.5 after CRF（68.2% for val）. Setting：DeepLab V2_resnet101 and other setting in voc12.yaml. So, the there is a mistack that i forget to use the CRF.

But I still have some problem.
1.In your paper,the mIOU using DeepLab V2 is 70.9% which is higher than mine. So , I wanna to konw your hyperparameter for the training deeplab-pytorch. 
 2.what your mean in last sentence'' the trained network for EPS is not the one we used in the paper''.So can you tell  me the trained networks in papar.

Thanks your detailed explanation.

**Comment by halbielee** [comment]
The difference can be the pseudo-masks for training the segmentation network.

For the clearance, we provide you the pseudo-masks we used in the paper. 
Here is [the link](https://drive.google.com/file/d/1JUux3TsejtWz5n__cxAB0-vwKMPYCA_4/view?usp=sharing).

Please train with the pseudo-masks we provide!

Thanks!!
